packages/web-react/src/types/shared/dictionaries.ts

```diff
@@ -16,6 +17,11 @@ export type AlignmentXDictionaryType<T = undefined> = (typeof AlignmentX)[Alignm
 export type ActionColorsDictionaryKeys = keyof typeof ActionColors;
 export type ActionColorsDictionaryType<T = undefined> = (typeof ActionColors)[ActionColorsDictionaryKeys] | T;
 
+export type ActionLinkColorsDictionaryKeys = keyof typeof ActionLinkColors;
+export type ActionLinkColorsDictionaryType<T = undefined> =
+  | (typeof ActionLinkColors)[ActionLinkColorsDictionaryKeys]
+  | T;
```

Comment:
I mean use the `C` in every type for Color.

Reply:
Done, check it